docs/netlify/README.md

```diff
@@ -0,0 +1,3 @@
+# Netlify DNS Provider
+
+TBD
```

Comment:
Please add the appropriate docu for the credentials in the secret and the optional provider config in the provider objects (if used)